IDEAS/Buildings/Components/BoundaryWall.mo

```diff
@@ -2,8 +2,11 @@ within IDEAS.Buildings.Components;
 model BoundaryWall "Opaque wall with boundary conditions"
   extends IDEAS.Buildings.Components.Interfaces.PartialOpaqueSurface(
      final QTra_design=U_value*AWall*(273.15 + 21 - TRef_a),
-     dT_nominal_a=-1);
-
+     dT_nominal_a=-1,
+    layMul(energyDynamics=energyDynamics));
+  parameter Modelica.Fluid.Types.Dynamics energyDynamics= if use_T_in then Modelica.Fluid.Types.Dynamics.DynamicFreeInitial else  Modelica.Fluid.Types.Dynamics.FixedInitial
```

Comment:
This default value can be changed by the user and therefore it would be better to add the `if` on line 6 such that:
```
layMul(energyDynamics=if use_T_in then Modelica.Fluid.Types.Dynamics.DynamicFreeInitial else  energyDynamics));
parameter Modelica.Fluid.Types.Dynamics energyDynamics= Modelica.Fluid.Types.Dynamics.FixedInitial
```